.github/workflows/integration.yml

```diff
@@ -254,20 +254,45 @@ jobs:
         repository: "${{ needs.configure.outputs.repo-name }}"
         persist-credentials: false
 
-    - name: Install Kubebuilder
-      run: |
-        version=2.3.1 # latest stable version
-        arch=amd64
-        curl -L -O "https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${version}/kubebuilder_${version}_linux_${arch}.tar.gz"
-        tar -zxvf kubebuilder_${version}_linux_${arch}.tar.gz
-        mv kubebuilder_${version}_linux_${arch} kubebuilder && sudo mv kubebuilder /usr/local/
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKER_USERNAME }}
+        password: ${{ secrets.DOCKER_PASSWORD }}
 
-    - name: Get dependencies for Test execution
-      run: |
-        go get -u github.com/ory/go-acc
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Cache Docker layers
```

Comment:
There is a step missing at the end, which moves the cache directory.

To prevent code duplication, I would check wether it is possible to add the building of the docker in the matrix and wait just for that execution (instead of the entire matrix)